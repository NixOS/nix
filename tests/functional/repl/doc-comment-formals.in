:l doc-comments.nix
"Note that this is not yet complete"
:doc documentedFormals
