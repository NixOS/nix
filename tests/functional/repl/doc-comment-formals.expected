Nix <nix version>
Type :? for help.

nix-repl> :l doc-comments.nix
Added <number omitted> variables.

nix-repl> "Note that this is not yet complete"
"Note that this is not yet complete"

nix-repl> :doc documentedFormals
Function `documentedFormals`\
  â€¦ defined at /path/to/tests/functional/repl/doc-comments.nix:57:5

Finds x
